cg/models/demultiplex/flow_cell.py

```diff
@@ -89,6 +93,15 @@ def sequencer_type(
         """Return the sequencer type."""
         return sequencer_types[self.machine_name]
 
+    @property
+    def mode(
```

Comment:
```suggestion
    def get_flow_cell_mode(
```

Reply:
Yes, no `get` for a property but what mode is it?
```suggestion
    def flow_cell_mode(
```

Reply:
or perhaps this was on a FlowCell? Disregard my rambling comment